gno.land/pkg/integration/testing_integration.go

```diff
@@ -369,6 +422,118 @@ func createAccount(env envSetter, kb keys.Keybase, accountName string) (gnoland.
 
 	return gnoland.Balance{
 		Address: address,
-		Amount:  std.Coins{std.NewCoin("ugnot", 1000000000000000000)},
+		Amount:  std.Coins{std.NewCoin("ugnot", 10e6)},
 	}, nil
 }
+
+type pkgsLoader map[string]gnomod.Pkg
+
+func (pkgs pkgsLoader) list() gnomod.PkgList {
+	list := make([]gnomod.Pkg, 0, len(pkgs))
+	for _, pkg := range pkgs {
+		list = append(list, pkg)
+	}
+	return list
+}
+
+func (pkgs pkgsLoader) Txs(creator bft.Address, fee std.Fee, deposit std.Coins) ([]std.Tx, error) {
+	pkgslist, err := pkgs.list().Sort()
+	if err != nil {
+		return nil, fmt.Errorf("uanble to sort packages: %w", err)
+	}
```

Comment:
Just curious, why do you need them sorted?

Reply:
Thank you for clarifying this 🙏 

cc @leohhhn @waymobetta

Reply:
It seems that a combination of a map for tracking visited packages and a list for maintaining the loading order did the trick. But people will still need to import theirs packages in the right order if they don't provide a `gno.mod`